WcaOnRails/app/controllers/persons_controller.rb

```diff
@@ -10,7 +10,9 @@ def index
           persons = persons.where("countryId = :region OR continentId = :region", region: params[:region])
         end
         if params[:search].present?
-          persons = persons.where("rails_persons.name LIKE :input OR wca_id LIKE :input", input: "%#{params[:search]}%")
+          params[:search].split.each do |part|
+            persons = persons.where("rails_persons.name LIKE ? OR wca_id LIKE ?", "%#{part}%")
```

Comment:
Can we share this code with the [`User.search`](https://github.com/cubing/worldcubeassociation.org/blob/392e961776ea0e56e5e94f97bcd41847f0a97140/WcaOnRails/app/models/user.rb#L519-L536) method? If we fix that method, it will also fix our uber search, which I think would be nice.